src/components/core/pivotTable/ResizedColumnsStore.ts

```diff
@@ -50,6 +66,12 @@ export class ResizedColumnsStore {
             return this.convertItem(this.manuallyResizedColumns[colId]);
         }
 
+        const weakColumnWidth = this.getMatchedWeakMeasuresColumnWidths(item);
```

Comment:
It seems to me that singular in the name of method would be more expressive, i.e. `getMatchedWeakMeasuresColumnWidth(..)` as it returns only one record

Reply:
done